### PR TITLE
location french created

### PR DESCRIPTION
**Pull request by mcman12**

### CODE REVIEW

File: contact-fr.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+title: Contact a11yYOW
+permalink: /contact/
+redirect_from:
```
Comment (nschonni) [inline contact-fr.html:5]: These pieces should prbably be removed sin't we don't want the old English URLs to redirect to the French. Same with the pages below

File: contact-fr.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+title: Contact a11yYOW
+permalink: /contact/
```
Comment (nschonni) [inline contact-fr.html]: What do you think of `/fr/contact/` as the link? Can't use `/contact/` because it would overwrite the English

**Comment by mcman12** [comment]
good catch. sorry my bad.
I'll get the rest translated tonight.

**Comment by nschonni** [comment]
Thanks @mcman12! I'lll look at how we can do the language switching and the navigation content

**Pull request merged**